_attributes/attributes-openshift-dedicated.adoc

```diff
@@ -27,3 +27,5 @@
 //Formerly known as CodeReady Containers and CodeReady Workspaces
 :openshift-local-productname: Red Hat OpenShift Local
 :openshift-dev-spaces-productname: Red Hat OpenShift Dev Spaces
+:hcp: Hosted Control Plane
```

Comment:
AFAIK, the name should be "hosted control planes" (no capitalization, and no abbreviation). In the OCP docs, the official name is "hosted control planes for Red Hat OpenShift Container Platform".  For ROSA, I imagine it should be something like "hosted control planes for ROSA".

Reply:
Is there a reason we can't abbreviate this? "hosted control planes" is a mouth-full, and even when reading it can increase cognitive load.

Reply:
@lahinson ☝️

Reply:
@wgordon17 Per the [official Red Hat product and solution names guidance](https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?usp=sharing), we are not supposed to abbreviate "hosted control planes". What I've found so far in the self-managed docs is that after hosted control planes is introduced, you're typically mentioning the hosted cluster or hosting cluster more than you're referring to "hosted control planes", so the name length hasn't been much of an issue.